test/sharness/t0044-add-symlink.sh

```diff
@@ -23,27 +27,34 @@ test_add_symlinks() {
 	'
 
 	test_expect_success "output looks good" '
-		echo QmWdiHKoeSW8G1u7ATCgpx4yMoUhYaJBQGkyPLkS9goYZ8 > filehash_exp &&
+		echo QmQRgZT6xVFKJLVVpJDu3WcPkw2iqQ1jqK1F9jmdeq9zAv > filehash_exp &&
 		test_cmp filehash_exp filehash_out
 	'
 
-	test_expect_success "adding a symlink adds the link itself" '
+	test_expect_success "adding a symlink adds the file itself" '
 		ipfs add -q files/bar/baz > goodlink_out
 	'
 
```

Comment:
I believe the intent of this test was really to add a symbolic link as there is limited support for them in the unixfs format.